__tests__/calculator.test.js

```diff
@@ -0,0 +1,17 @@
+import calculator from '../src/calculator';
+
+test('function returns sum of two numbers', () => {
+  expect(calculator.add(2, 4)).toBe(6);
```

Comment:
- [ ] [OPTIONAL] Let's test this with different numbers too. At least two test cases.